nextflow_schema.json

```diff
@@ -706,6 +706,49 @@
             },
             "fa_icon": "fas fa-clone"
         },
+        "contamination_estimation_for_human_dna": {
+            "title": "Contamination estimation for Human DNA",
+            "type": "object",
+            "description": "Options for the estimation of contamination of human DNA",
+            "default": "",
+            "fa_icon": "fas fa-radiation-alt",
+            "properties": {
+                "run_contamination_angsd": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-power-off",
+                    "description": "Turn on nuclear contamination estimation for human genomes with ANGSD.",
+                    "help_text": "Specify to run the optional processes for (human) nuclear DNA contamination estimation with ANGSD."
+                },
+                "angsd_chrom_name": {
+                    "type": "string",
+                    "default": "X",
+                    "description": "The name of the X chromosome in your bam/FASTA header. 'X' for hs37d5, 'chrX' for HG19.",
+                    "help_text": "The name of the human chromosome X in your bam. 'X' for hs37d5, 'chrX' for HG19.",
+                    "fa_icon": "fas fa-address-card"
+                },
+                "angsd_mapq": {
+                    "type": "integer",
+                    "default": 30,
+                    "description": "Specify the minimum mapping quality reads should have for contamination estimation with angsd.",
+                    "help_text": "> Modifies angsd parameter: `-minMapQ`",
+                    "fa_icon": "fas fa-thumbs-up"
```

Comment:
These should probably be consistent with other tools using MapQ/BaseQ filtering.